docs/development.rst

```diff
@@ -27,31 +30,31 @@ and new projects that align with consortium goals.
 
 `Documentation and tutorial curators` help maintain the documentation and tutorials.
 This can be as easy as pointing out confusing bits of the documentation in a
-GitHub `issue <http://github.com/popgensims/stdpopsim/issues>`_., or adding or editing
+GitHub `issue <http://github.com/popgensims/stdpopsim/issues>`__., or adding or editing
```

Comment:
I changed all external links in the docs to the `...`__ format with double underscore. This is apparently more robust to changes in the link text.